src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @include flex(center, '');
```

Comment:
The second parameter in the `flex` mixin is an empty string. If this parameter is optional, consider removing it or providing a valid value according to your mixin definition.